content/people/peymant.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Peyman Tehrani
+role: student-phd
+status: Ph.D. Student
+yearjoined: 2017
+portrait: peymant.jpg

```

Comment:
Rename ‘peymant.jpeg’ to ‘peymant.jpg’. Make sure that only the
‘peymant.jpg’ and ‘peymant-badge.jpg’ files remain.

Reply:
The wrong photo is kept in ‘peymant.jpg’. Please put the cropped
portrait into that file.

Reply:
Sorry: cropping the image it got renamed with different extension.

Reply:
No worries. Looks good now.